playbooks/kube-init.yml

```diff
@@ -0,0 +1,77 @@
+---
+- import_playbook: ka-init/init.yml
+- import_playbook: ka-lab-ipv6/ipv6-lab.yml
+  when: ipv6_enabled
+
+- hosts: master,nodes
+  become: true
+  become_user: root
+  tasks:
+    - name: Set bridge-nf-call-iptables to 1 (yes, this is redundant, earlier steps may not have been sticky)
+      shell: >
+        echo "1" > /proc/sys/net/bridge/bridge-nf-call-iptables
+      ignore_errors: true
+
+- hosts: master
+  become: true
+  become_user: root
+  # pre_tasks:
+  #   - debug:
+  #       msg: "Skip init? {{ skip_init }}"
+  roles:
+    - { role: kube-init }
+    # - { role: kube-template-cni }
+
+- hosts: master
+  tasks: []
+  roles:
+    - { role: kube-cni }
+
+- hosts: master
+  tasks: []
+  roles:
+    - { role: multus-crd, when: "pod_network_type == 'multus' and multus_use_crd and not multus_npwg_demo and not skip_init"}
```

Comment:
This will read a whole lot better when you use `include_role` and can add the `when` clauses as a multiline list.